src/test/java/step1/StringCharAtTest.java

```diff
@@ -0,0 +1,32 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+@DisplayName("String charAt() 테스트")
+public class StringCharAtTest {
+
+    @Test
+    @DisplayName("index가 주어졌을때 문자열의 index에 있는 문자를 반환한다.")
+    public void shouldReturnChar_givenIndex(){
+        String input = "abc";
+        int index = 0;
+        char output = input.charAt(index);
+        char expectedOutput = 'a';
+
+        assertThat(output).isEqualTo(expectedOutput);
```

Comment:
```suggestion
        char output = "abc".charAt(0);
        assertThat(output).isEqualTo('a');
```

모든 값을 로컬 변수로 구현하지 말고 위와 같이 하드코딩하는 것은 어떨까?
특히 테스트 코드에서는 하드코딩해도 되지 않을까?